services/rook-ceph-cluster/1.15.5/objectbucketclaims/helmrelease.yaml

```diff
@@ -34,3 +34,30 @@ spec:
       name: rook-ceph-cluster-cluster-overrides
       optional: true
   targetNamespace: ${releaseNamespace}
+---
+# This ingress is needed to make Traefik configure an additional route
+# for handling HTTPS requests to Ceph Object Store on the same port from
+# external clients such as:
+# - Velero CLI running external to K8S Clusters (management and attached).
+# - Kubecost pods running internal to K8S Clusters (attached only).
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: external-ceph
+  namespace: ${releaseNamespace}
+  annotations:
+    kubernetes.io/ingress.class: kommander-traefik
```

Comment:
This annotation is now deprecated. Lets use `ingressClassName` property of `spec`.

Reply:
fixed in [eb62c5b](https://github.com/mesosphere/kommander-applications/pull/3004/commits/eb62c5b4a1ad38b583d876d5d0a295ee9a1a2e55)